test/rules/invariant_booleans.dart

```diff
@@ -237,3 +237,13 @@ void bug372(bool foo) {
     // doSomethingElse();
   }
 }
+
+void bug337(int offset, int length) {
+  if (offset >= length) {
+    return;
+  }
+
+  offset++;
```

Comment:
This checks the postfix increment operator, but the rule checks for all four combinations of (prefix | postfix) (increment | decrement) operators. There are three operators that are not being tested, and I think they should be.